pkg/tarsum/tarsum_spec.md

```diff
@@ -10,6 +10,11 @@ This document describes the algorithms used in performing the TarSum checksum
 calculation on filesystem layers, the need for this method over existing
 methods, and the versioning of this calculation.
 
+## Warning
+
+This checksum algorithm is for best-effort comparision of file trees with fuzzy logic.
```

Comment:
Oh! "comparision"? Sorry, didn't spot that when reading on my phone earlier :sweat_smile: